apps/Better Chromium/uninstall

```diff
@@ -1,5 +1,16 @@
 #!/bin/bash
 
+if [[ "$1" != "update" ]];then
+  # rpi ubuntu (22.04+) can use Pi OS packaged chromium for better hardware support
+  if [ -f /etc/apt/sources.list.d/pi-apps-coders-chromium.sources ]; then
+    warning "When installing Better Chromium, you switched to PiOS chromium build. If you want to revert to the snap version, here's commands to do that."
+    warning '~/pi-apps/api rm_external_repo "pi-apps-coders-chromium" "force"'
+    warning 'sudo rm -f /etc/apt/preferences.d/pi-apps-coders-chromium'
+    warning 'sudo apt install --only-upgrade chromium-*'
+    warning 'sudo apt purge libwidevinecdm0'
```

Comment:
Do you intend for these 5 lines to all be preceded by a flashing yellow triangle and "WARNING: "?